orangecontrib/text/widgets/owconcordance.py

```diff
@@ -151,6 +155,21 @@ def matching_docs(self):
         else:
             return 0
 
+    def get_data(self):
+        domain = Domain([], metas=[StringVariable("Conc. {}".format(
+            self.word)), StringVariable("Document")])
+        data = []
+        docs = []
+        for row in range(self.rowCount()):
+            data.append([])
+            docs.append([])
+            for column in range(self.columnCount()):
+                index = self.index(row, column)
+                data[row].append(str(self.data(index)))
+            data[row] = [" ".join(data[row])]
+            docs[row] = [self.corpus.titles[self.word_index[row][0]]]
+        return(Corpus.from_list(domain, np.column_stack((data, docs))))
```

Comment:
`from_list` is inherited from `Table` and doesn't correctly initializes `Corpus`. Use `from_table`